composeApp/src/commonMain/kotlin/org/ooni/engine/Engine.kt

```diff
@@ -96,7 +96,10 @@ class Engine(
                         charging = isBatteryCharging(),
                         onWiFi = networkTypeFinder() == NetworkType.Wifi,
                         platform = platformInfo.platform.value,
-                        runType = taskOrigin.value,
+                        runType = when (taskOrigin) {
+                            TaskOrigin.AutoRun -> "timed"
+                            TaskOrigin.OoniRun -> "manual"
+                        },
```

Comment:
We should probably make this some type of property of the `TaskOrigin` >> `taskOrigin.runType` mabe.

Reply:
```kotlin
fun TaskOrigin.runType(): String {
    return when (this) {
        TaskOrigin.AutoRun -> "timed"
        TaskOrigin.OoniRun -> "manual"
    }
}
```